config/settings.yml

```diff
@@ -19,3 +19,10 @@ plugins:
     default: 1
     enum: "TrustLevelSetting"
     client: true
+  flag_posts_voting_comments_allowed_groups:
```

Comment:
Should we make `min_trust_to_flag_posts_voting_comments` hidden?

Reply:
Yep will do :)